projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/SupportedPythonVersions.java

```diff
@@ -134,4 +135,17 @@ public String getDefaultPythonVersion() {
   private String getBuilderImage(Map<String, String> supportedPythonVersion) {
     return supportedPythonVersion.getOrDefault(BUILDER_IMAGE, dockerRegistryService.builderImage());
   }
+
+  public boolean isVdkVersionPassedDifferentFromOneSetByPythonVersion(JobDeployment jobDeployment) {
```

Comment:
This logic needs to be moved to JobImageDeoloyer, since it is not related to supported versions.

Reply:
Moved. Thanks